src/test/java/org/opentripplanner/transit/speed_test/SpeedTest.java

```diff
@@ -76,10 +74,6 @@ public SpeedTest(
     this.config = config;
     this.transitModel = transitModel;
 
-    var features = new OtpConfigLoader(opts.rootDir()).loadOtpConfig();
-    OTPFeature.enableFeatures(features.otpFeatures);
-    OTPFeature.logFeatureSetup();
-
```

Comment:
These lines need to be moved to before calling `loadGraph` in `main`

Reply:
Sorry, was a bit quick - it was not the SpeedIntegration test that needed a fix, but the Flex IT. 

Should be ok now. I added a commit witch reorder the methods in the speed test - the second commit only reorder, nothing else.